### ISSUE TITLE
Paragraph in Styled numbered sections has not margin

### ISSUE THREAD

**Issue by linawolf** [TYPO3-Documentation/sphinx_typo3_theme#172]
This leads to not space between different paragraphs and no space between paragraphs and code-boxes:

![image](https://user-images.githubusercontent.com/48202465/185785786-cf0998b4-19da-4fd2-8adf-dde81f58c532.png)

This can be fixed by removing the following CSS:

```
.rst-content li>p {
  /* margin: 0; */
}
```

Additionally there is no margin at the bottom of the code box, which is caused by:

```
.bignums>li *:last-child, .bignums-hint>li *:last-child, .bignums-note>li *:last-child, .bignums-caution>li *:last-child, .bignums-warning>li *:last-child, .bignums-attention>li *:last-child, .bignums-important>li *:last-child, .bignums-seealso>li *:last-child, .bignums-tip>li *:last-child, .bignums-danger>li *:last-child, .bignums-error>li *:last-child, .bignums-xxl>li *:last-child {
  /* margin-bottom: 0; */
}
```

If I remove both these settings from the CSS the example looks much better readable:

![image](https://user-images.githubusercontent.com/48202465/185785975-b7a48246-a378-4243-9108-fea7965ec4e6.png)

The rest used in this example was:
```

4.  The namespace and PSR-4 autoloading

    TYPO3 features :ref:`PSR-4 autoloading <autoload>`. If a PHP class is
    withing the correct namespace and path it will be available automatically.

    Now what is the correct namespace for your extension? Look for the
    keyword "autoload" in the :file:`composer.json` of your extension. You
    will find something like this:

    ..  code-block:: json
        :caption: EXT:speeddemo/composer.json
        :emphasize-lines: 7

        {
            "name": "typo3-documentation-team/speeddemo",
            "type": "typo3-cms-extension",
            // ...
            "autoload": {
                "psr-4": {
                    "Typo3DocumentationTeam\\Speeddemo\\": "Classes/"
                }
            },
        }

    The key in the array "psr-4" is your namespace:
    :php:`Typo3DocumentationTeam\Speeddemo`. Note: the backspace needs to be
    escaped by another backspace in json.
```

**Comment by marble** [comment]
Please always leave a link. Nevertheless, investigating...

**Comment by marble** [comment]
Solved with sphinx_typo3_theme v4.7.7 and DRC v3.0.dev28
Now looking like so:
https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/ExtensionArchitecture/Tutorials/MinimalExtension.html
![image](https://user-images.githubusercontent.com/307057/188571953-a7d5aace-537c-4f5e-91f6-3765bcd775df.png)
